<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/neurotessmesh.ico</normaloff>:/icons/neurotessmesh.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenBlueConfig"/>
    <addaction name="actionOpenXMLScene"/>
    <addaction name="actionOpenSWCFile"/>
    <addaction name="actionCloseData"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionUpdateOnIdle"/>
    <addaction name="actionShowFPSOnIdleUpdate"/>
    <addaction name="actionWireframe"/>
    <addaction name="actionRenderOptions"/>
    <addaction name="actionEditSave"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionRecorder"/>
    <addaction name="actionAdvancedRecorderOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenBlueConfig"/>
   <addaction name="actionOpenXMLScene"/>
   <addaction name="actionOpenSWCFile"/>
   <addaction name="separator"/>
   <addaction name="actionHome"/>
   <addaction name="actionConfiguration"/>
   <addaction name="actionEditSave"/>
   <addaction name="actionWireframe"/>
   <addaction name="actionCloseData"/>
   <addaction name="actionRenderOptions"/>
   <addaction name="actionRecorder"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionHome">
    <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/home.png</normaloff>:/icons/rsc/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionUpdateOnIdle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update on idle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRenderOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/colorpicker.png</normaloff>:/icons/rsc/colorpicker.png</iconset>
   </property>
   <property name="text">
    <string>Render options</string>
   </property>
   <property name="toolTip">
    <string>Render options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionOpenBlueConfig">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/open_bc.png</normaloff>:/icons/rsc/open_bc.png</iconset>
   </property>
   <property name="text">
    <string>Open BlueConfig</string>
   </property>
   <property name="toolTip">
    <string>Open BlueConfig</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionOpenXMLScene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/open_xml.png</normaloff>:/icons/rsc/open_xml.png</iconset>
   </property>
   <property name="text">
    <string>Open XML Scene</string>
   </property>
   <property name="toolTip">
    <string>Open XML Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionCloseData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/close.png</normaloff>:/icons/rsc/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Data</string>
   </property>
   <property name="toolTip">
    <string>Close Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpenSWCFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/open_swc.png</normaloff>:/icons/rsc/open_swc.png</iconset>
   </property>
   <property name="text">
    <string>Open SWC</string>
   </property>
   <property name="toolTip">
    <string>Open SWC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionShowFPSOnIdleUpdate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show FPS on idle update</string>
   </property>
   <property name="toolTip">
    <string>Show FPS on idle update</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/wireframe.png</normaloff>:/icons/rsc/wireframe.png</iconset>
   </property>
   <property name="text">
    <string>Show wireframe</string>
   </property>
   <property name="toolTip">
    <string>Show wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionEditSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/reconstruction.png</normaloff>:/icons/rsc/reconstruction.png</iconset>
   </property>
   <property name="text">
    <string>Edit And Save</string>
   </property>
   <property name="toolTip">
    <string>Edit And Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/settings.png</normaloff>:/icons/rsc/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
   <property name="toolTip">
    <string>Configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRecorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rsc/recorder.svg</normaloff>:/icons/rsc/recorder.svg</iconset>
   </property>
   <property name="text">
    <string>Recorder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAdvancedRecorderOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced recorder options</string>
   </property>
   <property name="toolTip">
    <string>Enable advanced recorder options</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
